test/infrastructure/kind/mapper.go

```diff
@@ -79,6 +79,28 @@ type Mapping struct {
 var preBuiltMappings = []Mapping{
 
 	// TODO: Add pre-built images for newer Kind versions on top
+	// Pre-built images for Kind v1.27.
```

Comment:
```suggestion
	// Pre-built images for Kind v0.27.
```
Same for the pre-existing comments below in this file